src/db.c

```diff
@@ -6216,15 +6212,7 @@ end_critical_section(&db_station_info_lock, "db.c:Station_data" );
  
 begin_critical_section(&db_station_info_lock, "db.c:Station_data" );
 
-// restore_position doesn't appear to work.  I get the dialog
```

Comment:
I personally think that comments that say "Tried this and it doesn't work" are useful. Next time someone is looking through this section of code it tells them something important. Burying it in Git logs means it probably won't get looked at again.

Reply:
I really disagree.  There are so many of these that they make the code really, really hard to read.

I think that nobody's reading them in the code, either.

Reply:
Also:  It is not particularly difficult to use git to dig through the history of a file and find when and where a certain line was inserted, or what the history of a particular section of code was.  These tools should be used in preference to making the code difficult to read.

Reply:
A comment to the effect of "I tried to do X by doing Y, but it didn't work" might be reasonable, but leaving broken code in the comments is just too much.  Especially when some of this broken code is 15-20 years old.  The block of code in this particular section of diff was commented out in commit 660d61f9 back in 2006.